CHANGELOG.md

```diff
@@ -41,6 +41,8 @@
 
 - Fixed lack of `instance_name` and `protocols` reconciliation for `KongPluginBinding` when reconciling against Konnect.
   [#1681](https://github.com/Kong/gateway-operator/pull/1681)
+- The `KonnectExtension` status is kept updated when the `KonnectGatewayControlPlane` is rotated.
```

Comment:
By rotated we mean removed and recreated? Can we be a little more specific in this changelog so that it's clear which use cases are fixed/improved?

Reply:
here it is 6466545e8c4fb9319086c7693aa30d28f6f476f7